includes/class-wp-job-manager-email-notifications.php

```diff
@@ -325,13 +307,16 @@ private static function get_job_detail_fields( WP_Post $job, $sent_to_admin, $pl
 	/**
 	 * Output email header.
 	 *
-	 * @param WP_Job_Manager_Email $email          Email object for the notification.
-	 * @param bool                 $sent_to_admin  True if this is being sent to an administrator.
-	 * @param bool                 $plain_text     True if the email is being sent as plain text.
+	 * @param string $email_notification_key  Email object for the notification.
```

Comment:
`s/object/key`

Reply:
Fixed these in 6dd46dd2d7c431b867df4ba281e3e9a1f2bc1277